### PR TITLE
Added android:launchMode configurability

### LINKED ISSUE #95: Question: how to set android:launchMode with cargo apk
Not sure if I'm just completely missing something but I can't seem to figure out how to set the `android:launchMode` from my cargo.toml using cargo apk. 

```
[package.metadata.android]
launch_mode = "singleTask" 
```
or 

```
[[package.metadata.android.activity_metadatas]]
name = "launchMode"
value = "singleTask"
```
don't seem to work.

Tried a bunch of different ways but can't get it to work. Is it even supported or, like I said, am I missing something. :)

### PR DESCRIPTION
**Pull request by maxded**
Fixes #95 - although it would nice to introduce a proper serialize + deserialize mechanism for generating the `AndroidManifest.xml`

Would love to see this released in `cargo-apk` immediately after as it's a must feature for a project that I'm working on. Thanks in advance! :)

### CODE REVIEW

**Review by msiglreith — APPROVED** [review]
thanks! please add an entry in the sample `toml` in the cargo-apk readme

**Comment by msiglreith** [comment]
for releases, feel free to open a PR which bumps versions (incl dependencies) otherwise I will create one tomorrow

**Comment by maxded** [comment]
Done!

File: cargo-apk/README.md [hunk 1]
```diff
@@ -43,6 +43,11 @@ icon = "@mipmap/ic_launcher"
 # Defaults to false.
 fullscreen = false
 
+# Set the instruction of how the activity hsould be launched
```
Comment (msiglreith) [inline cargo-apk/README.md:46]:
```suggestion
# Set the instruction of how the activity should be launched
```

File: ndk-build/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # Unreleased
```
Comment (msiglreith) [inline ndk-build/CHANGELOG.md:1]:
```suggestion
# 0.1.3 (2020-11-21)
```

**Review by msiglreith — CHANGES_REQUESTED** [review]
A few small points left then good to go.
Also please add a changelog entry to `cargo-apk` for 0.5.5 to be clear what changed (in the dependencies)

File: cargo-apk/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,9 @@
 # Unreleased
 
+# 0.1.3 (2020-11-21)
```
Comment (msiglreith) [inline cargo-apk/CHANGELOG.md:3]: 0.5.5

**Review by msiglreith — APPROVED** [review]

**Pull request merged**